bpf/cpu/cpu.bpf.c

```diff
@@ -655,7 +668,7 @@ int walk_user_stacktrace_impl(struct bpf_perf_event_data *ctx) {
 
       previous_rsp = unwind_state->sp + 8 + ((((unwind_state->ip & 15) >= threshold)) << 3);
     } else {
-      bpf_printk("\t[error] register %d not valid (expected $rbp or $rsp)", found_cfa_type);
+      bpf_printk("\t[warn] register %d not valid (expected $rbp or $rsp)", found_cfa_type);
```

Comment:
"downgraded" these two logs as they are well-known limitations of our unwinder. They happen rarely enough too.